### PR TITLE
Single product page

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Added individual product page - https://deploy-preview-7--lucidbook-store.netlify.app/pages/productpage
*Added address page - https://deploy-preview-7--lucidbook-store.netlify.app/pages/address.html
*Added profile page  - https://deploy-preview-7--lucidbook-store.netlify.app/pages/profile.html

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -40,7 +39,15 @@
   overflow: hidden;
   height: 45rem;
 }
-/* utils  */
-.has-transparent {
-  background-color: transparent;
+.features {
```
Comment (ZuberDunge) [inline css/homepage.css:42]: suggestion : add space after {}

**Review by ZuberDunge — APPROVED** [review]
I saw the preview and the UI looks awesome, Code structure is excellent too, just add spaces after each {}, code will look even better. overall everything is fine. Good to go from my side.

**Review by RakshithBhat03 — APPROVED** [review]
The code looks clean and well structured. UI looks amazing. Good to go from my side.

File: css/homepage.css (hunk 1 above)
Comment (AnuragSahu11) [reply css/homepage.css:42]: Okay I will keep in mind from now on

**Pull request merged**